src/main/java/com/jaquadro/minecraft/storagedrawers/client/renderer/TileEntityDrawersRenderer.java

```diff
@@ -140,17 +114,12 @@ private void renderFastItemSet (TileEntityDrawers tile, BlockState state, PoseSt
 
             ItemStack itemStack = drawer.getStoredItemPrototype();
             renderStacks[i] = itemStack;
-            renderAsBlock[i] = isItemBlockType(itemStack);
-        }
-
-        for (int i = 0; i < drawerCount; i++) {
```

Comment:
Cool, I think the separation of block and item loops came from a time when they were rendered under different immediate mode settings.  A rendering optimization that hasn't been applicable for years.